components/list/list-item-checkbox-mixin.js

```diff
@@ -159,6 +167,7 @@ export const ListItemCheckboxMixin = superclass => class extends SkeletonMixin(s
 			<d2l-selection-input
 				@d2l-selection-change="${this._onCheckboxChange}"
 				?disabled="${this.selectionDisabled}"
+				.disabledTooltip=${this.selectionDisabledTooltip}
```

Comment:
```suggestion
				.disabledTooltip="${this.selectionDisabledTooltip}"
```

Reply:
I typically don't use quotes when I'm setting a property, but I don't feel strongly either way.

Reply:
🤷‍♂️ ... I always thought we added them, at least for consistency, but I see the Lit docs hardly ever include them anymore.